pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -570,6 +574,15 @@ func UnqualifiedSearchRegistries(ctx *types.SystemContext) ([]string, error) {
 	return config.UnqualifiedSearchRegistries, nil
 }
 
+// GetCredsStore return the list of credetntial store configed by creds_store for registries
+func GetCredsStore(ctx *types.SystemContext) ([]string, error) {
```

Comment:
No, the API should, if at all possible, give the user the effective configuration, not the raw pieces.

Reply:
What are effective configurations? Just want this API to export the `CredsStore` field to pkg/config/config.go

Reply:
If we export that field _individually_, we can’t easily change it or replace it; compare how we now can’t return a value exactly as documented for `ConfigDirPath` 

Instead, give the user an API to ask a specific question and receive a single response, like `FindRegistry` that returns effective configuration for a single input (which might never be physically present in any config file), or something like `getRegistryCredConfigs` in this case.

We might _eventually_ need to export the raw data, but let’s not commit to that API until there’s a clear need.